var/spack/repos/builtin/packages/dyninst/package.py

```diff
@@ -72,10 +81,19 @@ def install(self, spec, prefix):
                     '-DLIBELF_INCLUDE_DIR=%s'   % join_path(
                         libelf.include, 'libelf'),
                     '-DLIBELF_LIBRARIES=%s'     % join_path(
-                        libelf.lib, 'libelf.so'),
-                    '-DLIBDWARF_INCLUDE_DIR=%s' % libdwarf.include,
-                    '-DLIBDWARF_LIBRARIES=%s'   % join_path(
-                        libdwarf.lib, 'libdwarf.so')]
+                        libelf.lib, 'libelf.so')]
```

Comment:
Another `.so` here.

Reply:
@adamjstewart Ok.  Fixed that.  I will check in after I fix the issue above based on your reply:
```
depends_on("elf@1", type='link', when='@develop')
```
Thanks 
Jim G